cmd/podman/untag.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/pkg/adapter"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	untagCommand cliconfig.UntagValues
+
+	_untagCommand = &cobra.Command{
+		Use:   "untag [flags] IMAGE [NAME...]",
```

Comment:
do `[flags]` still hold?  I don't see a mention below about it in the md.

Reply:
No I think we can remove it since `untag` in specific has no flags.

Reply:
That works for me, thanks @saschagrunert

Reply:
So I'm very confused at to why the linter is requiring '[flags]` here is we have not flags?  It's a very confusing usage message.

Reply:
The integration tests parse this to automatically do some iterative testing:
https://github.com/containers/libpod/blob/c1a7911f37d5b9791be68f3a9e6478771d565731/test/system/015-help.bats#L45-L63

I could probably work around this somehow.